dp_wizard/app/results_panel.py

```diff
@@ -33,89 +33,7 @@ def button(name: str, ext: str, icon: str, primary=False):
     )
 
 
-def results_ui():
```

Comment:
Most of the details remain the same, but since at the top-level it is now conditional, it has to be in the server.